staging/src/k8s.io/kube-aggregator/hack/register-all-apis-from.sh

```diff
@@ -53,38 +53,40 @@ AGG_KUBECONFIG=${4}
 caBundle=$(base64 /var/run/kubernetes/server-ca.crt | awk 'BEGIN{ORS="";} {print}')
 
 # if we have a /api endpoint, then we need to register that
-if kubectl --kubeconfig=${FROM_KUBECONFIG} get --raw / | grep -q /api/v1; then
+if kubectl --kubeconfig="${FROM_KUBECONFIG}" get --raw / | grep -q /api/v1; then
 	group=""
 	version="v1"
 	resourceName=${version}.${group}
-	resourceFileName=${dir}/${resourceName}.yaml
-	cp ${scriptDir}/apiservice-template.yaml ${resourceFileName}
-	${SED} -i "s/RESOURCE_NAME/${resourceName}/" ${resourceFileName}
-	${SED} -i "s/API_GROUP/${group}/" ${resourceFileName}
-	${SED} -i "s/API_VERSION/${version}/" ${resourceFileName}
-	${SED} -i "s/SERVICE_NAMESPACE/${SERVICE_NAMESPACE}/" ${resourceFileName}
-	${SED} -i "s/SERVICE_NAME/${SERVICE_NAME}/" ${resourceFileName}
-	${SED} -i "s/CA_BUNDLE/${caBundle}/" ${resourceFileName}
+	resourceFileName="${dir}/${resourceName}.yaml"
+	cp "${scriptDir}/apiservice-template.yaml" "${resourceFileName}"
+	${SED} -i "s/RESOURCE_NAME/${resourceName}/" "${resourceFileName}"
+	${SED} -i "s/API_GROUP/${group}/" "${resourceFileName}"
+	${SED} -i "s/API_VERSION/${version}/" "${resourceFileName}"
+	${SED} -i "s/SERVICE_NAMESPACE/${SERVICE_NAMESPACE}/" "${resourceFileName}"
+	${SED} -i "s/SERVICE_NAME/${SERVICE_NAME}/" "${resourceFileName}"
+	${SED} -i "s/CA_BUNDLE/${caBundle}/" "${resourceFileName}"
 	echo "registering ${resourceName} using ${resourceFileName}"
 
-	kubectl --kubeconfig=${AGG_KUBECONFIG} create -f ${resourceFileName}
+	kubectl --kubeconfig="${AGG_KUBECONFIG}" create -f "${resourceFileName}"
 fi
 
-groupVersions=( $(kubectl --kubeconfig=${FROM_KUBECONFIG} get --raw / | grep /apis/ | sed 's/",.*//' | sed 's|.*"/apis/||' | grep '/') )
```

Comment:
why this change?

Reply:
this line hit this https://www.shellcheck.net/wiki/SC2207 .